src/megaapi_impl.cpp

```diff
@@ -6371,12 +6371,34 @@ bool MegaApiImpl::createPreview(const char *imagePath, const char *dstPath)
     fsAccess->path2local(&utf8DstPath, &localDstPath);
 
     sdkMutex.lock();
-    bool result = gfxAccess->savefa(&localImagePath, GfxProc::PREVIEW1000x1000, &localDstPath);
+    bool result = gfxAccess->savefa(&localImagePath, 1000, 1000, &localDstPath);
     sdkMutex.unlock();
 
     return result;
 }
 
+bool MegaApiImpl::createAvatar(const char *imagePath, const char *dstPath)
+{
+    if (!gfxAccess)
+    {
+        return false;
+    }
+    
+    string utf8ImagePath = imagePath;
```

Comment:
Should we protect `imagePaht` / `dstPath` being `NULL` ? it's directly received from the app.